tests/sentry/backup/README.md

```diff
@@ -0,0 +1,37 @@
+# Backup tests
+
+This directory tests a number of scenarios where we take an empty database, import a given backup `.json` file, optionally perform some transform, re-export it and validate the diff. The expected result is that only the changes we have made are reflected in the final diff between the original input `.json` that we imported, and the final output `.json` that we exported.
+
+## Comparators
```

Comment:
This is something you made up? It doesn't come from `jsondiff` or whatever?

Reply:
The comparators part is. `jsondiff` is just a naive JSON diffing framework - it takes two JSON blobs, compares them character-for-character, and tries its very best to give you a reasonable set of steps to achieve the diff (which updates, deletions, re-orderings etc you'd need to do on the left side to match the right). But like we talked about previously, there are some parts of a valid export that won't match it's companion import on a byte-wise basis, like `date_updated` timestamps. Using custom comparators ahead of `jsondiff` allows us to smooth over those expected and correct differences.

Reply:
Okay, can we add some verbiage to that effect? Suggestion in a new comment below ...

Reply:
Used the language you suggested below verbatim. Marking resolved.